docs/DeveloperGuide.md

```diff
@@ -719,6 +719,12 @@ _Some user stories are to be implemented beyond v1.4_
      * 5e2. User adds the given role to the McScheduler as a new role (UC-012).
  
        Use case resumes at step 4.
+       
+* 5f. The worker is already assigned to the shift under a role.
+ 
+     * 5f1. McScheduler shows an error message.
+     
+       Use case ends.
```

Comment:
Think this should be use case resumes at step 4? User can reassign or choose another shift/worker.
```suggestion
       Use case resumes at step 4.
```

Reply:
The use case is for assigning a worker to a shift so I assumed it would end when the worker is already assigned